exercises/src/main/scala/fpinscala/gettingstarted/GettingStarted.scala

```diff
@@ -36,7 +36,13 @@ object MyModule {
 
   // Exercise 1: Write a function to compute the nth fibonacci number
 
-  def fib(n: Int): Int = ???
+  def fib(n: Int): Int = {
+    def helper(n: Int, prev: Int = 0, acc: Int = 1): Int = {
```

Comment:
If you add `@annotation.tailrec` on helper, you'll have the compiler validate that the function is handled as tail recursive by the compiler. Otherwise, your function might not be stack safe